[XNNPACK][QS8] torch.nn.ELU

mcr229 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #104800
* __->__ #104307
* #104306
* #104305
* #104304

Differential Revision: [D47075933](https://our.internmc.facebook.com/intern/diff/D47075933/)

mcr229 committed:
[XNNPACK][QS8] torch.nn.ELU



Differential Revision: [D47075933](https://our.internmc.facebook.com/intern/diff/D47075933/)

[ghstack-poisoned]

mcr229 committed:
Update on "[XNNPACK][QS8] torch.nn.ELU"


Differential Revision: [D47075933](https://our.internmc.facebook.com/intern/diff/D47075933/)

[ghstack-poisoned]

mcr229 committed:
Update on "[XNNPACK][QS8] torch.nn.ELU"


Differential Revision: [D47075933](https://our.internmc.facebook.com/intern/diff/D47075933/)

[ghstack-poisoned]

vadimkantorov commented on torch/ao/quantization/backend_config/executorch.py:
```diff
@@ -346,6 +346,7 @@ def _get_share_qparams_ops_configs() -> List[BackendPatternConfig]:
     ]
     share_qparams_ops = [
         F.adaptive_avg_pool2d,
+        F.elu,
```
also a bit strange that here F.leaky_relu_ is with inplace variant and F.relu/F.elu is without

mcr229 committed:
Update on "[XNNPACK][QS8] torch.nn.ELU"


Differential Revision: [D47075933](https://our.internmc.facebook.com/intern/diff/D47075933/)

[ghstack-poisoned]
